brave-cxf3/src/main/java/com/github/kristofa/brave/cxf3/BraveClientFeature.java

```diff
@@ -0,0 +1,27 @@
+package com.github.kristofa.brave.cxf3;
+
+import com.github.kristofa.brave.Brave;
+import org.apache.cxf.Bus;
+import org.apache.cxf.feature.AbstractFeature;
+import org.apache.cxf.interceptor.InterceptorProvider;
+
+/**
+ * Configures cxf client with brave interceptors.
+ */
+public class BraveClientFeature extends AbstractFeature {
```

Comment:
eventhough you've hid the constructor, finaling public types is a nice reinforcement